### ISSUE TITLE
Bug: panic on receiving maliciously small packets

### ISSUE THREAD

**Issue by Banyc** [Matrix-Zhang/tokio_kcp#28]
There is an assertion on kcp.rs:46:5 from crate `kcp`:

```rust
/// Read `conv` from raw buffer
pub fn get_conv(mut buf: &[u8]) -> u32 {
    assert!(buf.len() >= KCP_OVERHEAD as usize);
```

When the receiver gets hit with a packet smaller than `KCP_OVERHEAD`, the whole process will just die.

I triggered this by simply `nc` the port with some random characters.

**Comment by Banyc** [comment]
@zonyitoo I bumped in this bug again and realized that this fix has not been in crates.io yet. It will be great to release it, huge thanks!

**Comment by zonyitoo** [comment]
released v0.9.5.
